File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Omsorgsgrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.PenPerson;
import no.nav.domain.pensjon.regler.kode.OmsorgTypeCti;

import java.io.Serializable;

public class Omsorgsgrunnlag implements Serializable {
    

    private int ar;
    private OmsorgTypeCti omsorgType;
    private PenPerson personOmsorgFor;
    private boolean bruk;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/Omsorgsgrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.PenPerson
import no.nav.pensjon.regler.domain.enum.OmsorgTypeEnum
import java.io.Serializable

class Omsorgsgrunnlag : Serializable {
    var ar = 0
    var omsorgTypeEnum: OmsorgTypeEnum? = null
    var personOmsorgFor: PenPerson? = null
    var bruk = false
}